src/zope/interface/tests/test_odd_declarations.py

```diff
@@ -258,7 +258,7 @@ class C(A, B):
         c.c
 
         import sys
-        if sys.version[0] == '2': # This test only makes sense under Python 2.x
+        if sys.version_info[0] < 3: # This test only makes sense under Python 2.x
```

Comment:
Rather than testing `sys.version_info`, this is probably better as a conditional import:

```python
        try:
            from types import ClassType
        except ImportError:
            pass
        else:
            assert not isinstance(C, (type, ClassType))

```